templates/test-environment-template.js

```diff
@@ -9,14 +13,15 @@ function makeTestEnvironment(_, simple) {
   const customActionStub = simple.stub();
 
   return {
-    _: _,
-    requireAccess: requireAccess,
-    requireRole: requireRole,
-    requireUser: requireUser,
-    channel: channel,
-    access: access,
-    role: role,
-    customActionStub: customActionStub,
+    _,
+    JSON,
+    requireAccess,
+    requireRole,
+    requireUser,
+    channel,
+    access,
+    role,
+    customActionStub,
```

Comment:
Switched to the shorthand form for object properties since this code is executed in a Node.js environment with access to ES2015.